server/services/offenderService.ts

```diff
@@ -2,6 +2,16 @@ import logger from '../../log'
 import { isNilOrEmpty, properCaseName } from '../utils/utils'
 import { AgencyId, OffenderService, PrisonerDetail, PrisonLocation } from '../types/uof'
 
+const compare = (a, b): number => {
+  let comparison = 0
+  if (a.userDescription.toUpperCase() > b.userDescription.toUpperCase()) {
+    comparison = 1
+  } else if (a.userDescription.toUpperCase() < b.userDescription.toUpperCase()) {
+    comparison = -1
+  }
+  return comparison
```

Comment:
you shouldn't need to create your own comparison function: you can use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)

Reply:
had this convo before... doh!